pdm.lock

```diff
@@ -49,48 +49,40 @@ files = [
 
 [[package]]
 name = "certifi"
-version = "2023.7.22"
```

Comment:
Q: Any way to do targeted update of only `jf_ingest`? Maybe I'm misremembering, but this feels like more changes than usual.